examples/basic-otlp/README.md

```diff
@@ -2,5 +2,92 @@
 
 This example shows basic span and metric usage, and exports to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) via OTLP.
 
-## Prerequisite
-You should first start a `opentelemetry-collector` on localhost using the default configuration. 
\ No newline at end of file
+## Usage
+
+```shell
+# Run `opentelemetry-collector`
+$ docker run  -p4317:4317 otel/opentelemetry-collector:latest
+
+# Report spans/metrics
+$ cargo run
+```
+
+# View result
+
+You should be able to see something similar below in the same console when you run docker
```

Comment:
It may be better to note that the time and id may be different.

Reply:
Updated in the latest version.